remote_power

```diff
@@ -14,17 +14,38 @@ reset_post_hook() {
 	[ "${TARGET}" = "bosch" ] && remote_usb bosch on
 }
 
-send_request() {
-	local res
-	local addr="$1"
+power_get_cmdline() {
+	# For example:
+	#       POWER_ADDR="tasmota-1234%5"
+	# becomes
+	#       POWER_CMDLINE="http://tasmota-1234/cm?cmnd=Power5"
+	#
+	local addr port
+
+	addr="${POWER_ADDR%\%*}"  # Everything before the '%'
+	case "${POWER_ADDR}" in  # If there is a '%',
+		*%*) port="${POWER_ADDR#*?\%}";;  # everything after it.
```

Comment:
Do we need the '?'. For me `${POWER_ADDR#*\%}` seems to work as well.

Reply:
Good question! It seems we need neither need it, nor the '\' following it, for things to work. (But we still need the '\' in the line above to distinguish from "%%".) Don't ask me where the '?' came from... Will fix!